src/drivers/barometer/ms5611/ms5611.cpp

```diff
@@ -173,12 +163,10 @@ class MS5611 : public cdev::CDev
 	/**
 	 * Initialize the automatic measurement state machine and start it.
 	 *
-	 * @param delay_ticks the number of queue ticks before executing the next cycle
-	 *
 	 * @note This function is called at open and error time.  It might make sense
 	 *       to make it more aggressive about resetting the bus in case of errors.
 	 */
-	void			start_cycle(unsigned delay_ticks = 1);
+	void			start_cycle();
```

Comment:
bmp280, mpl3115a2, and ms5611 are the only drivers that use `start_cycle()` rather than `start()`... but they don't have to be changed.